requirements/build.txt

```diff
@@ -1,12 +1,12 @@
 # Requirements for building wheels
 # This does not depend on runtime stuff so we do not
 # include base.txt
-sphinx
-sphinx-intl
-sphinx-rtd-theme
-sphinx-autobuild
+sphinx==2.4.4
```

Comment:
I think we need to pin it to 1.8.5 since starting 2.0.0, it dropped python 2.7 support. https://github.com/sphinx-doc/sphinx/blob/v2.0.0/CHANGES#L38

```suggestion
sphinx==1.8.5
```

Reply:
thank you